tests/h/activity/query_test.py

```diff
@@ -251,6 +252,25 @@ def test_it_only_shows_annotations_from_default_authority(self,
         AuthorityFilter.assert_called_once_with(pyramid_request.default_authority)
         search.append_modifier.assert_any_call(AuthorityFilter.return_value)
 
+    @pytest.mark.parametrize('enable_flag', [(True, False)])
+    def test_it_recognizes_wildcards_in_uri_url(self,
+                                                pyramid_request,
+                                                search,
+                                                UriFilter,
+                                                UriCombinedWildcardFilter,
+                                                enable_flag):
+
+        pyramid_request.feature.flags['wildcard_search_on_activity_pages'] = enable_flag
+
+        execute(pyramid_request, MultiDict(), self.PAGE_SIZE)
+
+        if enable_flag:
+            UriCombinedWildcardFilter.assert_called_once_with(pyramid_request)
+        else:
+            UriFilter.assert_called_once_with(pyramid_request)
```

Comment:
This branch never gets executed. Have a look at the codecov output and also the decorator on this test function.

Reply:
fixed!